### PR TITLE
Update share-calendars-with-external-users.md

### PR DESCRIPTION
**Pull request by msbemba**
Added sharing intervals to the document -reference https://support.microsoft.com/en-us/office/calendar-sharing-in-microsoft-365-b576ecc3-0945-4d75-85f1-5efafb8a37b4?ui=en-us&rs=en-us&ad=us

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7047

File: microsoft-365/admin/manage/share-calendars-with-external-users.md [hunk 1]
```diff
@@ -42,6 +42,18 @@ You can enable calendar sharing for all users in your organization in the Micros
 
 4. Choose what type of calendar information to make available to users. You can allow all information, or limit it to time only or time, subject, and location only.
 
+## External Sharing Sync Interval
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/share-calendars-with-external-users.md]:
```suggestion
## External sharing sync interval
```

File: microsoft-365/admin/manage/share-calendars-with-external-users.md [hunk 2]
```diff
@@ -42,6 +42,18 @@ You can enable calendar sharing for all users in your organization in the Micros
...
+
+Instant syncing for sharing outside your tenant isn't supported currently. While you can share in these configurations, syncing will happen periodically. There are two types of cross-tenant sharing:
+
+1) Microsoft 365 to another Microsoft 365 user (if external sharing is enabled).
+
+A full shared calendar is created, but the sync will happen approximately every three hours. Instant syncing will eventually be enabled for this setup.
+
+2) Microsoft 365 to an Outlook.com user. If external sharing is disabled, sharing to another Microsoft 365 user also falls into this group.
+
+An ICS URL is generated when sharing, which the recipient can use to add to any calendar service. With an ICS subscription, the recipient’s calendar service chooses when to sync the ICS subscription to receive new updates. If the recipient is an Outlook.com or an Microsoft 365 user, the sync will happen approximately every three hours.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/share-calendars-with-external-users.md]:
```suggestion
1. **Microsoft 365 to another Microsoft 365 user (if external sharing is enabled)**: A fully shared calendar is created, but the sync will happen approximately every three hours. Instant syncing will eventually be enabled for this setup.

2. **Microsoft 365 to an Outlook.com user**: If external sharing is disabled, sharing to another Microsoft 365 user also falls into this group. An ICS URL is generated when sharing, which the recipient can use to add to any calendar service. With an ICS subscription, the recipient’s calendar service chooses when to sync the ICS subscription to receive new updates. If the recipient is an Outlook.com or a Microsoft 365 user, the sync will happen approximately every three hours.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestions.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@msbemba Please resolve the conflict

**Pull request merged**